apps/platform-app/prisma/schema.prisma

```diff
@@ -56,6 +56,7 @@ model VerificationToken {
 
 model Profile {
   id                       String            @id @default(uuid())
+  slug                     String            @unique
```

Comment:
**issue:** Slug must be optional, we won't be able to apply such migration

Reply:
this would be handled by #369